### DISCUSSION TITLE
Notification delay for "Chat notification" plugin

### DISCUSSION THREAD

**Discussion by pettrs** [runelite/runelite#16147] [Ideas]
Is it possible to add a notification delay on chat notifications? the implemented chat notification in the Runelite settings doesn't apply to chat notifications so one would have to be added directly into the plugin.

The reason I'm requesting this is that I want to set a custom notification in the GOTR minigame to be notified 10 seconds prior to the first altar opening. while there is a notification when it opens, I would have to set my notification prior to it spawning, so when the minigame starts and then delay that notification for 110 seconds

![image](https://user-images.githubusercontent.com/75093127/212430534-d7dd35ce-dac3-4f94-a263-5dcb03204c7a.png)
(the timer I'm referring to)

Additionally, this would allow for more customizable notification methods.

Thank for the consideration

**Comment by Enriath** [comment]
At least for this case, you may be better off asking for this to be added to the GotR hub plugin. I'm not sure how many uses there are outside of that ngl, nor how the UX for it would look (unless it's just a general delay box like Idle Notifier has, which I imagine would annoy some people)
